src/componenets/modal/AddAtFolder.tsx

```diff
@@ -1,12 +1,22 @@
+import { SyntheticEvent } from "react";
 import close from "../../images/_close.png";
 
-function AddAtFolder({ toggleModal }) {
+type AddAtFolderType = {
+  toggleModal: (id: number) => void;
+};
+
+function AddAtFolder({ toggleModal }: AddAtFolderType) {
+  function handleImageClick(event: SyntheticEvent) {
```

Comment:
컴포넌트내의 함수는 일반적으로 화살표함수로 작성해주는 편이에요~
```suggestion
  const handleImageClick = (event: SyntheticEvent) => {
```